src/startupscriptgenerator/dotnetcore/main.go

```diff
@@ -44,6 +44,16 @@ func main() {
 		fullAppPath = common.GetValidatedFullPath(*appPathPtr)
 	}
 
+	// If a user supplies the repo root (instead of output folder), we try to see if the repo root has
+	// the default oryx publish output folder
+	defaultPublishOutputDirPath := filepath.Join(fullAppPath, "oryx_publish_output")
```

Comment:
consolidate `oryx_publish_output` to a shared constant

Reply:
also, this would only work if the user didn't specify a custom output directory, correct?